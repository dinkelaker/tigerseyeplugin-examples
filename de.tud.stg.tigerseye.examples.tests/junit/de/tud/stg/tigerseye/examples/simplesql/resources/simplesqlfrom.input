	def q12 = SELECT "NAME","AGE" FROM "PERSONS"	
