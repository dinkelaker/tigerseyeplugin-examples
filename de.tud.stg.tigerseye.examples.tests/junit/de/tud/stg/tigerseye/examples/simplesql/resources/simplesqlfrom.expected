def q12 = selectFrom(
[
"NAME",
"AGE"
] as String[],
[
"PERSONS"
] as String[])